.travis.yml

```diff
@@ -100,14 +100,22 @@ install:
         requests
         jsonschema
   - source activate test-environment
+  # install additional packages through conda-forge, wherever possible, since
+  # it seems a bit more stable than pip which sometimes fails with what looks
+  # like network errors
+  - conda config --add channels conda-forge
+  # "--no-update-dependencies" to not change versions of already installed packages
+  # github3.py is dependency of obspy_github_api
+  - conda install --no-update-dependencies
+        pep8-naming
+        github3.py
   # install packages not available via conda
   - pip install codecov
   - pip install geographiclib
   # current pyimgur stable release has a py3 incompatibility
   - pip install https://github.com/megies/PyImgur/archive/py3.zip
+  - pip install https://github.com/obspy/obspy_github_api/archive/0.5.0.zip
```

Comment:
How about just always installing the master? Then this would not have to be changed all the time and changes to API helper get directly used. This is only for our internal use so it should be fine.